commit 2b871f4dbffe3801d0da3f89806b5935f758d5f3
Author: Jia Tan <jiat0218@gmail.com>
Date:   Wed Aug 9 20:55:36 2023 +0800

    Docs: Update INSTALL for --enable-threads method win95.
    
    The Autotools build allows win95 threads and --enable-small together now
    if the compiler supports __attribute__((__constructor__)).

diff --git a/INSTALL b/INSTALL
index 89742ab7..787ee5fa 100644
--- a/INSTALL
+++ b/INSTALL
@@ -466,8 +466,10 @@ XZ Utils Installation
                         win95   Use Windows 95 compatible threads. This
                                 is compatible with Windows XP and later
                                 too. This is the default for 32-bit x86
-                                Windows builds. The `win95' threading is
-                                incompatible with --enable-small.
+                                Windows builds. Unless the compiler
+                                supports __attribute__((__constructor__)),
+                                the `win95' threading is incompatible with
+                                --enable-small.
 
                         vista   Use Windows Vista compatible threads. The
                                 resulting binaries won't run on Windows XP